composables/useGraphql.ts

```diff
@@ -26,7 +26,7 @@ export const useQueryParams = ({
 export default function ({
   queryPrefix = '',
   queryName,
-  clientName = '',
+  clientName = computed(() => ''),
```

Comment:
It could stay the same as it does not make any different.
```suggestion
  clientName = '',
```

Reply:
this throw warning if not computed

Reply:
ok. Let's move on